src/legacy/core_plugins/kibana/public/dashboard/panel/panel_state.ts

```diff
@@ -96,22 +89,17 @@ function findTopLeftMostOpenSpace(width: number, height: number, currentPanels:
       }
     }
   }
-  return { x: 0, y: Infinity };
+  return { x: 0, y: maxY };
```

Comment:
This was actually a bug that I caught in my embeddable PR, I'm not sure how it surfaced, but the jest tests I added catches it and this fixes it.